python/python/tests/test_async_client.py

```diff
@@ -2113,7 +2113,7 @@ async def test_zmscore(self, redis_client: TRedisClient):
 
     @pytest.mark.parametrize("cluster_mode", [True, False])
     @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
-    async def test_zinterstore(self, redis_client: TRedisClient):
+    async def test_zinter_commands(self, redis_client: TRedisClient):
```

Comment:
You don't have to merge the tests, because `zinter` is much easier to test rather than `zinterstore`